util_file.py

```diff
@@ -93,8 +93,38 @@ def ls_remote(user, machine, directory, port=22):
 
     command = f"ls {directory}"
     stdin, stdout, stderr = ssh.exec_command(command)
+    stdout = stdout.read().decode(errors='ignore'), stderr.read().decode(errors='ignore')
 
-    return stdout.read().decode(errors='ignore'), stderr.read().decode(errors='ignore')
+    return stdout
+
+
+def find_remote(user, machine, directory, serach, port=22):
```

Comment:
should this be custom_search, instead of serach?

Reply:
as above.